acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -1,7 +1,7 @@
 import frictionless from '../../scripts/frictionless.js';
 import { redirectLegacyBrowsers } from '../../scripts/legacyBrowser.js';
 
-const pageLang = document.querySelector('html').lang;
+const pageLang = document.querySelector('html').lang || 'en-US';
```

Comment:
This is a workaround. The lang attribute is missing this early. We need another way to pass in the locale.

Reply:
Yes, that will happen before the ROW rollout